modules/local/appendmetadata/main.nf

```diff
@@ -0,0 +1,19 @@
+process WRITE_METADATA {
```

Comment:
Could you remove this file, since it was moved to the `writemetadata/` directory.

Reply:
Thanks :)